### PR TITLE
Remove SB parent from master

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by geoand** [comment]
Although I have updated all boosters, I am only opening a PR for this one to start with.

**Comment by geoand** [comment]
There seems to be an error with parent  pom. I will fix it shortly

File: pom.xml [hunk 1]
```diff
@@ -17,10 +17,151 @@
...
+        <artifactId>arquillian-bom</artifactId>
+        <version>${arquillian.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <version>${junit.version}</version>
+        <scope>import</scope>
```
Comment (metacosm) [inline pom.xml]: This is wrong: scope should be `test` but I see you've fixed that in the scripts.

Comment (geoand) [reply pom.xml]: Yup.

I updated the scripts and I am soon going to commit the updates to the boosters

**Comment by geoand** [comment]
Some fixes have been made, should be in better shape now

**Comment by geoand** [comment]
retest this please

**Comment by geoand** [comment]
@Ladicek Could you please take a look and see if the Openshift cluster that runs the tests is OK? 
The error messages from the tests are not making much sense so I wonder if there is something wrong

File: pom.xml [hunk 2]
```diff
@@ -5,22 +5,184 @@
...
+      <version>22</version>
     </parent>
 
     <artifactId>spring-boot-cache-parent</artifactId>
     <version>1.5.13-3-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <name>Spring Boot - Cache Booster</name>
+    <groupId>io.openshift.booster</groupId>
+  <name>Spring Boot - Cache Booster</name>
```
Comment (gytis) [inline pom.xml]: Indentations are inconsistent through the whole file.

File: pom.xml [hunk 3]
```diff
@@ -5,22 +5,184 @@
...
+        <scope>test</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.jayway.restassured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <version>2.9.0</version>
```
Comment (gytis) [inline pom.xml]: Move to property.

File: pom.xml [hunk 4]
```diff
@@ -5,22 +5,184 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.jayway.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <version>1.7.0</version>
```
Comment (gytis) [inline pom.xml]: Move to property.

File: pom.xml [hunk 5]
```diff
@@ -5,22 +5,184 @@
...
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
+  <profiles>
+
+  <profile>
+    <id>licenses</id>
```
Comment (gytis) [inline pom.xml]: This profile is not needed.

File: pom.xml (hunk 2 above)
Comment (geoand) [reply pom.xml]: This is the result of the updates being done in an automated fashion.
Once all the issues relating to functionality are fixed, I will go through the poms and manually fix such issues

File: pom.xml (hunk 5 above)
Comment (geoand) [reply pom.xml]: I'll get rid of it for all boosters

File: pom.xml (hunk 4 above)
Comment (geoand) [reply pom.xml]: Will do for all boosters

**Comment by geoand** [comment]
@gytis I addressed your comments

**Comment by geoand** [comment]
retest this please

**Comment by geoand** [comment]
retest this please

**Comment by metacosm** [comment]
We're missing several things: `developers` section (though I'm not sure how important that is, up to the team), `java.version` and related `maven.compiler` properties.

File: pom.xml [hunk 6]
```diff
@@ -5,22 +5,171 @@
...
+      <artifactId>booster-parent</artifactId>
+      <version>22</version>
     </parent>
 
     <artifactId>spring-boot-cache-parent</artifactId>
     <version>1.5.13-3-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <name>Spring Boot - Cache Booster</name>
+    <groupId>io.openshift.booster</groupId>
```
Comment (metacosm) [inline pom.xml]: This should be grouped close to the `artifactId` if possible.

**Comment by geoand** [comment]
I didn't add the `developers` section because I thought that it's not really relevant in this case.

As for the versions, I thought that the things defined in the booster-parent would be enough:

https://github.com/openshiftio/booster-parent/blob/210dedc37f87ffb99aa105254c5e2382d62e3176/pom.xml#L33
https://github.com/openshiftio/booster-parent/blob/210dedc37f87ffb99aa105254c5e2382d62e3176/pom.xml#L39

File: pom.xml [hunk 7]
```diff
@@ -5,22 +5,171 @@
...
+    <plugins>
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
```
Comment (metacosm) [inline pom.xml]: Don't we need the `licenses` profile as well? Or it that obsolete, @gytis?

Comment (geoand) [reply pom.xml]: @gytis mentioned in one of the other PRs that it is no longer needed

**Comment by metacosm** [comment]
Ah yes, missed the `java.version` from the parent… so we're all good there.

File: pom.xml (hunk 6 above)
Comment (geoand) [reply pom.xml]: That's what happens when things get automated :wink: 
I go over the PRs again and manually fix such issues

File: pom.xml (hunk 7 above)
Comment (metacosm) [reply pom.xml]: Nevermind, I see that it's not needed anymore :)

File: pom.xml (hunk 6 above)
Comment (metacosm) [reply pom.xml]: No need for such a small issue.

File: pom.xml (hunk 7 above)
Comment (gytis) [reply pom.xml]: It's not needed. We're using licenses-generator project for that.

**Comment by gytis** [comment]
I don't think we should add developers section to the boosters as they are used as a basis to generate a client application. Client for whom this application is generated should be in developers section.

**Pull request merged**